rust/src/internal/mod.rs

```diff
@@ -220,3 +220,34 @@ pub extern "C" fn opendp_internal___extrinsic_distance(
     let descriptor = try_!(to_str(descriptor)).to_string();
     Ok(AnyMetric::new(ExtrinsicDistance { descriptor })).into()
 }
+
+#[bootstrap(
+    features("contrib"),
+    arguments(function(rust_type = "$pass_through(TO)")),
+    dependencies("c_function")
+)]
+/// Construct a Function from a user-defined callback.
+/// This is meant for internal use, as it does not require "honest-but-curious",
+/// unlike `new_function`.
+///
+/// See `new_function` for correct usage and proof definition for this function.
```

Comment:
Because this is bootstrapped, would this show up in the python API docs? In that context, it would be nice if we could link to `new_function`, though I don't think we show any proof definitions in the docs.

Reply:
I don't think it will show up in python API docs because it's in a private module.